docs/content/reference/values.txt

```diff
@@ -434,6 +434,7 @@
 |gloo.deployment.podSecurityContext.seccompProfile.localhostProfile|string|||
 |gloo.deployment.podSecurityContext.appArmorProfile.type|string|||
 |gloo.deployment.podSecurityContext.appArmorProfile.localhostProfile|string|||
+|gloo.deployment.podSecurityContext.seLinuxChangePolicy|string|||
```

Comment:
could we add a description here?

Reply:
Hmm. We don't have any description for the pod security context stuff and it likely is just a mirror of kuebrnetes fields.